app/build.gradle

```diff
@@ -49,6 +49,8 @@ dependencies {
     implementation libraries.androidxCoreKtx
     implementation libraries.androidxMaterial
     implementation libraries.androidxRecyclerView
+    debugImplementation libraries.chucker
+    releaseImplementation libraries.chuckerNoOp
```

Comment:
chucker is cool. it lets you see the request/responses via the notification shade.